conda-forge.yml

```
@@ -1,14 +1,19 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
+  linux_64: linux_64
+  # NOTE: testing on aarch64 and ppc64le fails with:
+  #   ImportError: /home/conda/[...]/site-packages/vtkmodules/../../.././././libGLdispatch.so.0: cannot allocate memory in static TLS block
+  # linux_aarch64: linux_64
+  # linux_ppc64le: linux_64
+  osx_64: osx_64
   osx_arm64: osx_64
```

no need to specify native build platforms; also, re-enable aarch/ppc
```suggestion
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
  osx_arm64: osx_64
```

---

sorry 🤦🏻